trestle/core/commands/author/jinja.py

```diff
@@ -204,20 +205,26 @@ def jinja_multiple_md(
         trestle_root: pathlib.Path,
         r_input_file: pathlib.Path,
         r_output_file: pathlib.Path,
-        profile: Optional[str],
+        profile_name: Optional[str],
         lut: Dict[str, Any],
         parameters_formatting: Optional[str] = None
     ) -> int:
         """Output profile as multiple markdown files using Jinja."""
         template_folder = pathlib.Path.cwd()
 
         # Output to multiple markdown files
-        profile_path = ModelUtils.full_path_for_top_level_model(trestle_root, profile, Profile)
+        profile, profile_path = ModelUtils.load_top_level_model(
+            trestle_root,
+            profile_name,
+            Profile,
+            FileContentType.JSON
+        )
```

Comment:
technically you don't need to specify json here, to allow flexibility on using yaml.

Reply:
done